### PR TITLE
feat(tests): AVModeration tests.

### PR DESCRIPTION
**Pull request by damencho**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -139,10 +139,10 @@
     "@types/amplitude-js": "8.16.5",
     "@types/audioworklet": "0.0.29",
     "@types/dom-screen-wake-lock": "1.0.1",
-    "@types/jasmine": "5.1.4",
     "@types/js-md5": "0.4.3",
     "@types/jsonwebtoken": "9.0.7",
     "@types/lodash-es": "4.17.12",
+    "@types/mocha": "10.0.10",
```
Comment (saghul) [inline package.json:145]: Why the change from jasmine to mocha?

**Review by saghul — APPROVED** [review]

**Comment by saghul** [comment]
After this PR, would you say the foundation is ready to start adding more tests?

File: package.json (hunk 1 above)
Comment (damencho) [reply package.json:145]: The skip method impl is missing in jasmine. And they don't plan to add it, there was an issue about it. 
I tried using what they advice to use (pending) but I couldn't make it work and to be as elegant as it is with mocha.

**Comment by damencho** [comment]
> After this PR, would you say the foundation is ready to start adding more tests?

Oh yes. I don't see anything else fundamental to be missing. I'm moving one by one the tests from torture in alphabetical order.

For the record old torture AVmoderation test takes 3min and 15 seconds while this one takes 2 minutes less :) I'm still amazed by the difference.

File: package.json (hunk 1 above)
Comment (saghul) [reply package.json:145]: Kewl!
